peach/src/main/java/com/hartwig/hmftools/peach/PeachUtils.java

```diff
@@ -0,0 +1,15 @@
+package com.hartwig.hmftools.peach;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class PeachUtils
+{
+    public static final Logger PCH_LOGGER = LogManager.getLogger(PeachApplication.class);
```

Comment:
Just FYI using a central logger this way does have the downside that you cannot tune logging based on class/package, and you will not see what class logs the message in your log entries.  Nothing wrong with what you've done, just wanted to mention.